data/json/recipes/recipe_food.json

```
@@ -3236,6 +3236,19 @@
     "//": "you're just melting the chocolate with the heat of the boiled milk, insignificant additional energy needed",
     "components": [ [ [ "milk_standard", 1, "LIST" ], [ "lf_milk_standard", 1, "LIST" ] ], [ [ "chocolate", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "milk_choc",
+    "id_suffix": "from_powder",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_DRINKS",
+    "skill_used": "cooking",
+    "time": "2 m",
+    "autolearn": true,
+    "//": "you're just dissolve the milk cocoa powder in the water. That take some time because cocoa powder have some fat. Therefore, you need to mix well",
```

```suggestion
    "//": "You're just dissolving the milk cocoa powder into the water. That takes some time because cocoa powder has some fat. Therefore, you need to mix well",
```
~~Typos in the player-invisible section reeee~~